ibis/mapd/tests/test_client.py

```diff
@@ -84,3 +84,35 @@ def test_union_op(alltypes):
     expr = t1.union(t2)
     with pytest.raises(com.UnsupportedOperationError):
         expr.compile()
+
+
+def test_create_table_schema(con):
+    t_name = 'mytable'
+
+    con.drop_table(t_name, force=True)
+
+    schema = ibis.schema([
+        ('a', 'float'),
+        ('b', 'double'),
+        ('c', 'int32'),
+        ('d', 'int64'),
+        ('x', 'point'),
+        ('y', 'linestring'),
+        ('z', 'polygon'),
+        ('w', 'multipolygon')
+    ])
+
+    try:
+        con.create_table(t_name, schema=schema)
```

Comment:
This needs to be outside the try, since the finally will still get executed even if this call fails. I do see the `force=True`, but you don't need that if you move this call outside the try.

Reply:
sounds good I will do it. thanks!